generators/entity/templates/server/src/main/java/package/domain/_Entity.java

```diff
@@ -406,7 +406,7 @@ public String toString() {
                 const fieldTypeBlobContent = fields[idx].fieldTypeBlobContent;
                 const fieldName = fields[idx].fieldName;
                 const fieldInJavaBeanMethod = fields[idx].fieldInJavaBeanMethod; _%>
-            ", <%= fieldName %>='" + <% if (fieldType.toLowerCase() === 'boolean') { %>is<% } else { %>get<%_ } _%><%= fieldInJavaBeanMethod %>() + "'" +
+            ", <%= fieldName %>=<% if (fieldType.toLowerCase() !== 'integer' && fieldType.toLowerCase() !== 'long' && fieldType.toLowerCase() !== 'float' && fieldType.toLowerCase() !== 'double' && fieldType.toLowerCase() !== 'bigdecimal') {%>'<%}%>" + <% if (fieldType.toLowerCase() === 'boolean') { %>is<% } else { %>get<%_ } _%><%= fieldInJavaBeanMethod %>() <% if (fieldType.toLowerCase() !== 'integer' && fieldType.toLowerCase() !== 'long' && fieldType.toLowerCase() !== 'float' && fieldType.toLowerCase() !== 'double' && fieldType.toLowerCase() !== 'bigdecimal') {%>+ "'" <%}%>+
```

Comment:
use `if (! ['integer', 'long',...].includes(fieldType.toLowerCase()) {` as its much more cleaner to read

Reply:
same in other places

Reply:
Sure will do, as well as creating a variable.

Thank you for the suggestion, it will be done as soon as possible

Reply:
It is fixed within commit 1a87ab2288014a6b050ce6e7cf65aa3f6c7fc38d